lib/kapnismology/smoke_test_run.rb

```diff
@@ -0,0 +1,33 @@
+require 'json'
+
+module Kapnismology
+  # A summary of a run containing smoke test evaluations
+  class SmokeTestRun
```

Comment:
Not really run anything, the evaluation is the runner. 
Maybe SmokeTestResponse?